modules/ibi-image-based-install-cluster-guide.adoc

```diff
@@ -15,6 +15,8 @@ For a successful image-based installation and deployment, see the following guid
 
 * If your cluster deployment at the edge of the network requires a proxy configuration, you must create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
 
+* Set the `clusterNetwork` and `serviceNetwork` network configurations in the seed cluster. The Lifecycle Agent embeds these settings in the seed image and they persist to the deployed cluster. You cannot change these network configurations later in the image-based installation and deployment process.
```

Comment:
The user doesn't need to explicitly set these values, they just need to be aware that the configuration present on the seed cluster persists to the deployed cluster and cannot be changed later.
Consider:
The clusterNetwork and serviceNetwork network configurations on the seed cluster persist to the deployed cluster. The Lifecycle Agent embeds these settings in the seed image, and they cannot be changed later in the image-based installation and deployment process.